app/utils/broadcast.py

```diff
@@ -301,3 +301,22 @@ def get_centroid_if_postcode_in_db(postcode, form):
         form.postcode.process_errors.append("Enter a postcode within the UK")
     else:
         return get_centroid(area)
+
+
+def format_area_name(area_name):
+    if area_name.endswith(", City of"):
+        return f"City of {area_name[:-9]}"
+    elif area_name.endswith(", County of"):
+        return f"County of {area_name[:-11]}"
+    else:
+        return area_name
+
+
+def format_areas_list(areas_list):
+    if isinstance(areas_list, CustomBroadcastAreas):
+        if len(areas_list) == 1:
+            return [format_area_name(area.name) for area in areas_list]
+        else:
+            return [format_area_name(area.name) for area in areas_list.items]
```

Comment:
This still doesn't work right?  What happens if you pass a CustomBroadcastAreas with 1 instance, it will still have to use items?

The only way you don't want to loop through the items is if it's a CustomBroadcastArea (singular, without an array).

This is why in my suggestion I said  to check against instance of CustomBroadcastAreas vs CustomBroadcastArea

Reply:
Removed the comparison to length being 1 and implemented the 'areas' v 'area' approach